packages/components/psammead-brand/src/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { string } from 'prop-types';
+import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-constants/breakpoints';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_HLF,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+} from '@bbc/gel-constants/spacings';
+
+const layoutWrapperWithoutGrid = css`
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_BELOW_400PX};
+  }
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const SVG_TOP_OFFSET = '1.25rem'; // 20px
+const SVG_BOTTOM_OFFSET = '1.5rem'; // 24px
+const BANNER_HEIGHT = '5rem'; // 80px
+
+const SVG_HEIGHT_PX = 24;
+const SVG_HEIGHT = `${SVG_HEIGHT_PX / 16}rem`;
+const SVG_WIDTH_PX = 167.95;
+const SVG_WIDTH = `${SVG_WIDTH_PX / 16}rem`;
+
+const VIEWBOX_VALUES = `0 0 ${SVG_WIDTH_PX} ${SVG_HEIGHT_PX}`;
+
+const StyledWrapper = styled.div`
+  ${layoutWrapperWithoutGrid};
+  background-color: ${C_POSTBOX};
+  height: ${BANNER_HEIGHT};
+  width: 100%;
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+const StyledLink = styled.a`
+  padding-top: ${GEL_SPACING};
+  display: inline-block;
+`;
+
+const StyledSpan = styled.span`
+  display: block;
+  padding-bottom: ${SVG_BOTTOM_OFFSET};
+  ${StyledLink}:hover &,
```

Comment:
```suggestion
  ${/* sc-selector */ StyledLink}:hover &,
  ${/* sc-selector */ StyledLink}:focus & {
```
By adding interpolation tagging here, this enables stylelint to know that `StyledLink` is a selector and will therefore lint the styles accordingly. 

Reference: https://github.com/styled-components/stylelint-processor-styled-components/blob/86b660f93018d6d77bf5e6af3b6e26db1630f106/README.md#interpolation-tagging
(Found via this PR https://github.com/styled-components/stylelint-processor-styled-components/pull/70) 

Then you can remove the line you've added to the `.stylelintrc` file.